Stripe/STPPaymentMethodType.m

```diff
@@ -65,15 +65,15 @@ @interface STPPaymentMethodTypeSEPADebit : STPPaymentMethodType
 @implementation STPPaymentMethodTypeSEPADebit
 
 - (UIImage *)paymentMethodImage {
-    return [STPImageLibrary addIcon];
+    return [STPImageLibrary sepaIcon];
 }
 
 - (UIImage *)paymentMethodTemplateImage {
-    return [STPImageLibrary addIcon];
+    return [STPImageLibrary sepaIcon];
 }
 
 - (NSString *)paymentMethodLabel {
-    return STPLocalizedString(@"New SEPA Debit", @"Label for 'new sepa debit' payment method field") ;;
+    return STPLocalizedString(@"Direct Debit Account", @"Label for SEPA Direct Debit payment method field") ;;
```

Comment:
ah yep, I can add a `paymentMethodAccessibilityLabel` property that maps to `paymentMethodLabel` by default